lib/std/json/hashmap.zig

```diff
@@ -0,0 +1,103 @@
+const std = @import("std");
+const Allocator = std.mem.Allocator;
+
+const ParseOptions = @import("static.zig").ParseOptions;
+const innerParse = @import("static.zig").innerParse;
+const innerParseFromValue = @import("static.zig").innerParseFromValue;
+const Value = @import("dynamic.zig").Value;
+const StringifyOptions = @import("stringify.zig").StringifyOptions;
+const stringify = @import("stringify.zig").stringify;
+const encodeJsonString = @import("stringify.zig").encodeJsonString;
+
+/// A thin wrapper around `std.StringArrayHashMapUnmanaged` that implements
+/// `jsonParse`, `jsonParseFromValue`, and `jsonStringify`.
+/// This is useful when your JSON schema has an object with arbitrary data keys
+/// instead of comptime-known struct field names.
+pub fn ArrayHashMapUnmanaged(comptime T: type) type {
```

Comment:
Suggest to omit "Unmanaged" since there is no "managed" variant. There is a precedent here with `MultiArrayList` and it's possible that the std lib may make unmanaged the default (or even not provide managed variants).

```suggestion
pub fn ArrayHashMap(comptime T: type) type {
```